test/lint/lint-python-utf8-encoding.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018-2022 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+# Make sure we explicitly open all text files using UTF-8 (or ASCII) encoding to
+# avoid potential issues on the BSDs where the locale is not always set.
+
+import sys
+import re
+
+from subprocess import check_output, CalledProcessError
+
+
+def check_fileopens():
+    fileopens = list()
+
+    try:
+        fileopens = check_output(["git", "grep", r" open(", "--", "*.py", ":(exclude)src/crc32c/"], universal_newlines=True, encoding="utf8").splitlines()
```

Comment:
```suggestion
        fileopens = check_output(["git", "grep", r" open(", "*.py", ":(exclude)src/crc32c/"], universal_newlines=True, encoding="utf8").splitlines()
```
I'm wondering if we can do this because they give the same output (locally tested).

Reply:
It's not strictly necessary here but adding the `--` is "safe". It's good to add just in case file arguments are added later that might conflict with option arguments.

Reply:
Sounds reasonable.